images/router/haproxy/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN INSTALL_PKGS="haproxy18" && \
     yum clean all && \
     mkdir -p /var/lib/haproxy/router/{certs,cacerts} && \
     mkdir -p /var/lib/haproxy/{conf,run,bin,log} && \
-    touch /var/lib/haproxy/conf/{{os_http_be,os_edge_http_be,os_tcp_be,os_sni_passthrough,os_reencrypt,os_route_http_expose,os_route_http_redirect,cert_config,os_wildcard_domain}.map,haproxy.config} && \
+    touch /var/lib/haproxy/conf/{{os_http_be,os_edge_reencrypt_be,os_tcp_be,os_sni_passthrough,os_route_http_redirect,cert_config,os_wildcard_domain}.map,haproxy.config} && \
```

Comment:
Ooh, I was just merging in these changes into my branch and I noticed the map is still called `os_edge_http_be` and not `os_edge_reencrypt_be` and it is still being referenced as "os_edge_http_be" in the use backend directives. So this PR needs changes.

Reply:
This is now done, but github hasn't noticed.